backend/models/time.py

```diff
@@ -8,6 +8,9 @@
 from backend.models import distance
 from backend.models.base import Measurer
 
+import openrouteservice
+from openrouteservice.distance_matrix import distance_matrix
```

Comment:
If this is literally the same as the MapBox API, could we possibly use that class instead?